docs_src/examples/Ch-ExamplesAddingModbusDevice.md

```diff
@@ -80,102 +77,264 @@ Once we execute a command, device-modbus knows its value type and
+  -
+    name: "DemandWindowSize"
+    description: "Demand window size in minutes; default is 15 min"
+    attributes:
+      { primaryTable: "HOLDING_REGISTERS", startingAddress: "4603" }
+    properties:
+      value:
+        { type: "UINT16", readWrite: "R", scale: "1"}
+      units:
+        { type: "String", readWrite: "R", defaultValue: "min"}
+  -
+    name: "LineFrequency"
+    description: "Line frequency setting for metering: 50=50 Hz, 60=60Hz"
+    attributes:
+      { primaryTable: "HOLDING_REGISTERS", startingAddress: "4609" }
+    properties:
+      value:
+        { type: "UINT16", readWrite: "R", scale: "1"}
+      units:
+        { type: "String", readWrite: "R", defaultValue: "Hz"}
+deviceCommands:
+  -
+    name: "Current"
+    get:
+      - { index: "1", operation: "get", deviceResource: "Current" }
+  -
+    name: "Values"
+    get:
+      - { index: "1", operation: "get", deviceResource: "Energy" }
+      - { index: "2", operation: "get", deviceResource: "Power" }
+      - { index: "3", operation: "get", deviceResource: "Voltage" }
+  -
+    name: "Configuration"
+    set:
+      - { index: "1", operation: "set", deviceResource: "DemandWindowSize" }
+      - { index: "2", operation: "set", deviceResource: "LineFrequency" }
+    get:
+      - { index: "1", operation: "get", deviceResource: "DemandWindowSize" }
+      - { index: "2", operation: "get", deviceResource: "LineFrequency" }
+coreCommands:
+  -
+    name: "Current"
+    get:
+      path: "/api/v1/device/{deviceId}/Current"
+      responses:
+        -
+          code: "200"
+          description: "Get the Current"
+          expectedValues: ["Current"]
+        -
+          code: "500"
+          description: "internal server error"
+          expectedValues: []
+  -
+    name: "Values"
+    get:
+      path: "/api/v1/device/{deviceId}/Values"
+      responses:
+        -
+          code: "200"
+          description: "Get the Values"
+          expectedValues: ["Energy","Power","Voltage"]
+        -
+          code: "500"
+          description: "internal server error"
+          expectedValues: []
+  -
+    name: "Configuration"
+    get:
+      path: "/api/v1/device/{deviceId}/Configuration"
+      responses:
+        -
+          code: "200"
+          description: "Get the Configuration"
+          expectedValues: ["DemandWindowSize","LineFrequency"]
+        -
+          code: "500"
+          description: "internal server error"
+          expectedValues: []
+    put:
+      path: "/api/v1/device/{deviceId}/Configuration"
+      parameterNames: ["DemandWindowSize","LineFrequency"]
+      responses:
+        -
+          code: "204"
+          description: "Set the Configuration"
+          expectedValues: []
+        -
+          code: "500"
+          description: "internal server error"
+          expectedValues: []
+```
 
 ### Set Up Device Service Configuration
 
 Use this configuration file to define devices and schedule jobs. The
```

Comment:
use `AutoEvent `instead of `schedule jobs` here

Reply:
Modified.